ui_framework/doc_site/src/views/description_list/description_list_styling.js

```diff
@@ -25,28 +21,17 @@ const favoriteVideoGames = [
 ];
 export default () => (
   <div>
-    <KuiFlexGroup>
-      <KuiFlexItem>
-        <KuiDescriptionList align="center" compressed listItems={favoriteVideoGames} />
-      </KuiFlexItem>
-      <KuiFlexItem>
-        <KuiDescriptionList type="column" align="center" compressed>
```

Comment:
Don't think you meant to delete this one. This version is important to show. It's the most common version for the align property actually, which is a column that right aligns the title, and left aligns the description. Right now you're just showing the row one and the inline one.